cmd/demo/payment.go

```diff
@@ -75,7 +75,9 @@ func (ch *paymentChannel) sendUpdate(update func(*channel.State), desc string) e
 	})
 	ch.log.Debugf("Sent update: %s, err: %v", desc, err)
 	if balChanged {
-		fmt.Println("💸 Sent payment. New balance:", state.Allocation.Balances[0])
+		bals := state.Allocation.Balances[0]
+		myBalEth, peerBalEth := weiToEther(bals[ch.Idx()]).String(), weiToEther(bals[1-ch.Idx()]).String()
+		fmt.Printf("💰 Sent payment. New balance: [My: %s Ξ, Peer: %s Ξ]\n", myBalEth, peerBalEth)
```

Comment:
Use variadic `weiToEth` and `%v` for `Stringer`s.

Reply:
Addressed in this commit - 806b792.